make.html.markdown

```diff
@@ -0,0 +1,236 @@
+---
+language: make
+contributors:
+    - ["Robert Steed", "https://github.com/robochat"]
+filename: Makefile
+---
+
+A Makefile defines a graph of rules for creating a target (or targets).
+It's purpose is the do the minimum amount of work needed to update a 
```

Comment:
Well it´s always going to be reviewed, otherwise you can fix errors also later :)